stream-legal-entity/legal-entity-bootstrap-task/pom.xml

```diff
@@ -20,7 +20,6 @@
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
-            <scope>provided</scope>
```

Comment:
am I missing anything here? for me it fails as class are not in classpath

Reply:
It don't think we need this dependency at all anymore